.github/workflows/upload-assets.yml

```diff
@@ -0,0 +1,80 @@
+name: Upload Packages to new release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    container:
+      image: innogames/graphite-ch-optimizer:builder
+
+    outputs:
+      deb: ${{ steps.build.outputs.deb }}
+      rpm: ${{ steps.build.outputs.rpm }}
+      sha256sum: ${{ steps.build.outputs.sha256sum }}
+      md5sum: ${{ steps.build.outputs.md5sum }}
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout
+        with:
+          # Otherwise there's a risk to not get latest tag
+          # We hope, that the current commit at
+          # least 50 commits close to the latest release
+          fetch-depth: 50
```

Comment:
risky

Reply:
the chance that the release has tag is ~100%, I'd say